cmd/debugmailbox/src/lib.rs

```diff
@@ -368,11 +371,18 @@ fn debugmailboxcmd(context: &mut ExecutionContext) -> Result<()> {
             )?;
 
             // Ask permission-slip to sign it.
-            let mut permslip = Process::new("permslip")
+            let mut permslip = Process::new("permslip");
+            permslip
                 .arg("sign")
                 .arg(key_name)
                 .arg("--kind=debug-authn-challenge")
-                .arg(format!("--beacon={beacon}"))
+                .arg(format!("--debug-authn-beacon={beacon}"));
```

Comment:
Nice catch! This changed in [permslip#164](https://github.com/oxidecomputer/permission-slip/pull/164).